project_organization.md

```diff
@@ -55,6 +55,26 @@ OTIAB members have the following primary responsibilities:
 - Provide feedback about possible priorities and/or specific proposals from the OTSC
 - Represent OpenTracing’s interests within their own organizations
 
+## OpenTracing Working Groups
+
+Working Group members meet at regularly to discuss and define the OpenTracing project structure, as well as perform day-to-day project management duties.
+
+### Language Working Group
+
+The Language Working Group manages the OpenTracing APIs and ecosystem. In order to handle the growing volume of contributions, this working groups is tasked with developing the protocols for proposing and submitting changes to the OpenTracing language APIs and instrumentation ecosystem. In addition, members of this Working Group triage the API and contrib backlogs, to ensure PRs and Issues are resolved in a timely manner.
+
+Any number of working group members are allowed; preferably two or more experts per language. Maintainers with overlapping responsibilities divide the work amongst themselves. PR and Issue standards are developed in a cross-language fashion. 
+
+#### Responsibilities
+- Schedule and attend Language Working Group sessions.
```

Comment:
these are per-language? e.g., if I work on C#, I don't care about PHP and don't want to attend that session since I may have nothing to add.

Reply:
Not every language has the same amount of contributors nor same enthusiasm. IMO one of the goals of having a **community** supporting languages is to spread and share the success working groups like java had. It can be that one could have nothing to add in a language conversation but I guess there are loads of stuff to add in other matters.

Reply:
I agree with @jcchavezs. Also, there actually is a great deal that we need to care about, cross language-wise, in order to provide a unified experience for OpenTracing users.